dotcom-rendering/package.json

```diff
@@ -36,12 +36,7 @@
 		"build-storybook": "node --max-old-space-size=4096 $(yarn bin)/build-storybook",
 		"makeBuild": "NODE_ENV=production CI_ENV=github webpack --config ./scripts/webpack/webpack.config.js"
 	},
-	"bundlesize": [
-		{
-			"path": "./dist/ga.*.js",
-			"maxSize": "21 kB"
-		}
-	],
+	"bundlesize": [],
```

Comment:
This is an aside but I don't think we're using the package `bundlesize` correctly?

Running `make bundlesize` and `yarn bundlesize` locally just returns reports of one random chunk per build:

```
 PASS  ./dist/ga.legacy.6562512b7657bf34b173.js: 3.31KB < maxSize 21KB (gzip) 
 PASS  ./dist/ga.modern.bfb6df33849398fbc36b.js: 2.77KB < maxSize 21KB (gzip) 
 PASS  ./dist/ga.variant.73f43741dc853772cc20.js: 2.82KB < maxSize 21KB (gzip) 
```

If we no longer use or rely on this then we can also remove the dependency and scripts from `makefile` and `ci-dcr.sh`

Reply:
@OllysCoding or @mxdvl might have more context on this

Reply:
A little bit of context as to why this was added in the first place:
https://github.com/guardian/dotcom-rendering/pull/727#discussion_r316307199

This does seem to suggest we don't use it at all, but maybe had the intention to.

If that's the case we should just remove it and add it when we know what we want to test with it so we don't need to engineer around it going forward.